contracts/upgradeable_contracts/erc20_to_native/HomeBridgeErcToNative.sol

```diff
@@ -119,6 +119,18 @@ contract HomeBridgeErcToNative is
         _setBlockRewardContract(_blockReward);
     }
 
+    /**
+     * @dev Withdraws the erc20 tokens or native coins from this contract.
+     * @param _token address of the claimed token or address(0) for native coins.
+     * @param _to address of the tokens/coins receiver.
+     */
+    function claimTokens(address _token, address _to) external onlyIfUpgradeabilityOwner {
+        // Since native coins are being minted by the blockReward contract and burned by sending them to the address(0),
+        // they are not locked at the contract during the normal operation.
+        // So, any erc20 token and native coins can be safely claimed.
+        claimValues(_token, _to);
```

Comment:
What will happen if native tokens are sent by the `selfdestruct` call? They will not be burnt. Does it mean that we need to provide an ability to claim the native tokens anyway?

Reply:
Yes, correct, and since we do not have a mechanism like `fixMediatorBalance` in vanilla bridges, the only option is to use the `claimTokens` functionality.

Reply:
Ok. Let's add a comment here describing that we are allowing to claim native tokens in order to handle this particular case when tokens are sent as part of minting tokens for coinbase account (this configuration is weird but it could be still) and as part of the `selfdesctruct` call -- in both cases our fallback function will not be called.